apps/sshtosnmp.c

```diff
@@ -124,21 +137,35 @@ main(int argc, char **argv) {
     {
         struct msghdr msg = { 0 };
         struct iovec iov = { buf, buf_len };
-        struct ucred ouruser = { 0 };
         struct cmsghdr *cmsg;
-	char cmsgbuf[CMSG_SPACE(sizeof(ouruser))];
```

Comment:
I'd like to preserve the `CMSG_SPACE(sizeof(ouruser))` expression for the Linux case. Please move the `cmsgbuf` declaration into the `#ifdef` statement such that it can be preserved.